src/fmu/tools/rms/copy_rms_param_to_ertbox_grid.py

```diff
@@ -482,37 +504,28 @@ def active_params_in_zone(
     number_layers: int,
     start_layer: int,
     end_layer: int,
-):
+) -> Any:
```

Comment:
Here you are returning a numpy array; be more spesific than "Nay"

Reply:
Fixed